### PR TITLE
Changes V3.1 - split preamble, notice, Mappings (Value-Only, Metadata, Path)

### PR DESCRIPTION
**Pull request by BirgitBoss**
index and preamble split + editorial changes + add notice chapter etc.

ValueOnly and other changes

### CODE REVIEW

**Comment by BirgitBoss** [comment]
@j-blume : may you please review this (small) change? Thank you

**Comment by BirgitBoss** [comment]
@mjacoby may you please review with respect to idShortPath and ValueOnly

File: documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_idShortPath.adoc [hunk 1]
```diff
@@ -12,9 +12,23 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur f
...
+To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an unnamed JSON-array. The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is present. In the first case, the submodel element's idShort is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[<Index>]" are appended after the idShort of the containing SubmodelElementList.
 
-In the following example, where a request for idShort paths starts at _MySubmodelElementCollection_ with SerializationModifier level = deep, the list of idShort paths is returned as follows:
+[.underline]#Grammar:#
+
+[listing]
+....
+<idShortPath> ::= <idShort> {["." <idShort> | "["<Index>"]" ]}*
```
Comment (mjacoby) [inline documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_idShortPath.adoc:21]: This grammar allows indices with leading zeros, e.g. 00000 or 0000012. Not sure if this is intentional and/or acceptable. An alternative grammar that does not allow these leading zeros could be
```<idShortPath> ::= <idShort> {["." <idShort> | "["<Index>"]" ]}*

<Index> ::= <Zero> | <PositiveNumber>

<PositiveNumber>  ::= <NonZeroDigit>{<Digit>}*
<Digit> ::= <Zero> | <NonZeroDigit>
<Zero>  ::= "0"
<NonZeroDigit>  ::= "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
```

File: documentation/IDTA-01001/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,2 +1,59 @@
 == Specification of the Asset Administration Shell
-= Part 1: Metamodel (IDTA-01001-3-1)
+= Part 1: Metamodel (IDTA-01001)
+
+====
+This specification is part of the https://industrialdigitaltwin.org/en/content-hub/aasspecifications[Asset Administration Shell Specification series]
```
Comment (mjacoby) [inline documentation/IDTA-01001/modules/ROOT/pages/index.adoc:5]:
```suggestion
This specification is part of the https://industrialdigitaltwin.org/en/content-hub/aasspecifications[Asset Administration Shell Specification series].
```

**Review by mjacoby — CHANGES_REQUESTED** [review]
Disclaimer: I did not review changes related to AsciiDoc syntax

Content-wise, this looks good, mostly small editorial things like typos.
The grammar for idShortPath could be improved to not allow leading zeros in array indices, a proposal has been included in the comments.

**Review by J-Blume — CHANGES_REQUESTED** [review]
Just one small addition to Michaels review since there was a missing "]". I also checked the ascidoc and it looks good.

**Comment by g1zzm0** [comment]
Adapt title of PR, because Value Only, PathType and MetaData Mappings are discussed

File: documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc [hunk 3]
```diff
@@ -99,19 +100,19 @@ Note: the examples are written in RDF/Turtle syntax, and only "Hello" and "Hallo
...
+Note 1: due to the limits in the representation of numbers in JSON, the maximum integer number that can be used without losing precision is 2³-1 (defined as Number.MAX_SAFE_INTEGER). Even if the used data type would allow higher or lower values, they cannot be used if they cannot be represented in JSON. Affected data types are unbounded numeric types xs:decimal, xs:integer, xs:positiveInteger, xs:nonNegativeInteger, xs:negativeInteger, xs:nonPositiveInteger and the bounded type xs:unsignedLong. Other numeric types are not affected. footnote:[cf. https://eclipse-esmf.github.io/samm-specification/2.1.0/datatypes.html (with adjustments for +/-INF, NaN, and language-typed literal support)]
```
[the remark this answers is not in the document]
Comment (BirgitBoss) [reply documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc:109]: Everthing needed is copied from ESMF. ESMF/SAMM is not an implementation of AAS. This is just the source of information.

[the remark this answers is not in the document]
Comment (BirgitBoss) [reply documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc:109]: Tahnks for the findings of the missing superscript.
I will fix the superscript in PR https://github.com/admin-shell-io/aas-specs/pull/375 (with asciidoc https://docs.asciidoctor.org/asciidoc/latest/text/subscript-and-superscript/ ) 2^53^-1

File: documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_idShortPath.adoc (hunk 1 above)
Comment (BirgitBoss) [reply documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_idShortPath.adoc:21]: will change

File: documentation/IDTA-01001/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,2 +1,59 @@
...
+SPDX-License-Identifier: CC-BY-4.0
+
+== How to Get in Contact
+
+Contact: https://industrialdigitaltwin.org/en/contact[IDTA]
+
+Sources: https://github.com/admin-shell-io/aas-specs[github]
+
+Feedback:
+* https://github.com/admin-shell-io/aas-specs/issues/new/choose[new issues, bugs
```
[the remark this answers is not in the document]
Comment (BirgitBoss) [reply documentation/IDTA-01001/modules/ROOT/pages/index.adoc]: ok

**Comment by BirgitBoss** [comment]
Value-Only will be discussed in further PR https://github.com/admin-shell-io/aas-specs/pull/375

**Comment by BirgitBoss** [comment]
findings were resolved, branch will be merged so that also the other can review it in html.

> Adapt title of PR, because Value Only, PathType and MetaData Mappings are discussed

done

**Pull request merged**